dev/archery/archery/benchmark/google.py

```diff
@@ -67,8 +67,7 @@ def results(self, repetitions=1):
 class GoogleBenchmarkObservation:
     """ Represents one run of a single (google c++) benchmark.
 
-    Observations are found when running with `--benchmark_repetitions`. Sadly,
-    the format mixes values and aggregates, e.g.
+    Aggregates are reported when running with `--repetitions`.
```

Comment:
```suggestion
    Aggregates are reported by Google Benchmark executables
    alongside other observations whenever repetitions are specified
    (with `--benchmark_repetitions` on the bare benchmark, or with
    the archery option `--repetitions`). Aggregate observations are
    not included in `GoogleBenchmark.runs`.
```

Reply:
I've updated the docstring with this text. I'm only noticing now, that I could have tried clicking the "Commit suggestion" button. I'll try that next time. Thanks!